packages/react-reconciler/src/ReactFiberHooks.old.js

```diff
@@ -120,7 +119,6 @@ type Update<S, A> = {|
   eagerReducer: ((S, A) => S) | null,
   eagerState: S | null,
   next: Update<S, A>,
-  priority?: ReactPriorityLevel,
```

Comment:
We used to use this for the Suspense priority warning. That got removed, but if we were to add the warning back, we'd use the update lanes.